test/integration/contentful_test.exs

```diff
@@ -21,7 +21,7 @@ defmodule Contentful.DeliveryTest do
     use_cassette "entry" do
       entry = Delivery.entry(@space_id, @access_token, "5JQ715oDQW68k8EiEuKOk8")
 
-      assert is_map(entry["fields"])
+      assert is_list(entry["fields"])
```

Comment:
Indeed, after some tests, the HTTP responses output objects and not arrays.
Apparently, the issue is within `resolve_includes` which changes the nature of the response. This might be a drawback of using Dict.

I'll investigate a bit further by tomorrow, but I'm open to suggestions for this one ;)